src/app/containers/MostRead/Canonical/index.jsx

```diff
@@ -56,19 +60,33 @@ const CanonicalMostRead = ({
 
   useEffect(() => {
     if (!items) {
-      const handleResponse = async response => {
+      const handleResponse = url => async response => {
         if (!response.ok) {
-          throw Error(response.statusText);
+          throw Error(
+            `Unexpected response (HTTP status code ${response.status}) when requesting ${url}`,
+          );
         }
         const mostReadData = await response.json();
         setItems(processMostRead({ data: mostReadData, numberOfItems }));
       };
 
       const fetchMostReadData = pathname =>
         fetch(pathname, { mode: 'no-cors' })
-          .then(handleResponse)
-          .catch(e => logger.error(`HTTP Error: "${e}"`));
+          .then(handleResponse(pathname))
+          .catch(error => {
+            logger.error(
+              JSON.stringify(
+                {
+                  event: MOST_READ_FETCH_ERROR,
+                  message: error.toString(),
+                },
+                null,
+                2,
```

Comment:
This is not quite right - should be 
``` 
logger.error(MOST_READ_FETCH_ERROR, {
error
});
```
or if you prefer:
``` 
logger.error(MOST_READ_FETCH_ERROR, {
message: error.toString()
});
```

Reply:
Updated this as suggested in the first block.

Reply:
I ended up updating this to the 2nd suggestion to follow the same convention as radio schedules

Reply:
Thanks, appreciate it.